pombola/core/seltests/basics.py

```diff
@@ -1,29 +1,45 @@
 from django.core.urlresolvers import reverse
 from pombola.testing.selenium import PombolaSeleniumTestCase
+from django.test.utils import override_settings
+from nose.plugins.attrib import attr
+from django_webtest import WebTest
+from mock import patch
 
-class CoreTestCase(PombolaSeleniumTestCase):
```

Comment:
Enabled this test case and replaced it with a `WebTest` because the Selenium tests aren't compatible with Django > 1.7.

It raises this exception: `AttributeError: 'module' object has no attribute 'commit_unless_managed'`